en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,188 @@
+
+The main purpose of a minor release is to ship new features and enhancements.
+
+- **Backward compatibility:** Yes - within their Silverstripe CMS major version series
+- **Cadence:** Every six months within the most recent stable Silverstripe CMS major version series. Minor releases are targeted for April and October each year
+- **Pre-release schedule:**
+  - **Alpha**: Typically not performed
+  - **Beta**: Approximately 6 weeks prior to the anticipated stable release. At this point, no new features or new public API should be added
+  - **RC**: Approximately 2 weeks prior to the stable release. At this point, only bug fixes should be added
+- **Documentation:** A changelog is published listing significant new features
+
+## Support lifecycle
+
+> [!NOTE]
+> For a detailed timeline of current and planned support durations for each Silverstripe CMS version, refer to the [support timeline](https://www.silverstripe.org/software/roadmap#support-timeline).
+
+A clear support lifecycle for each Silverstripe CMS version is defined. This provides long-term stability and planning for users.
+
+At any given time, only two major version series are officially supported. For example, CMS 5.4, CMS 6.0, and CMS 6.1 can all be supported at once as they span a total of two major version series, however, CMS 5.4, CMS 6.0, and CMS 7.0 cannot all be supported at once.
+
+Each Silverstripe CMS version has a *standard* support length, except for the last minor version of a Silverstripe CMS major version series, which has an  *extended* support length, e.g. CMS 5.4 has extended support. Details about how this affects release timelines is detailed further below.
+
+Support commitments for security and bug fixes are met through patch releases on individual supported modules and are detailed further in a section below.
+
+Each Silverstripe CMS version progresses through three defined support phases.
+
+### Full support
+
+A Silverstripe CMS version enters the *full support* phase after the corresponding release has been performed.
+
+- **Security fixes:** Yes – security vulnerabilities at all impact levels are addressed
+- **Bug fixes:** Yes - provided they do not change existing public API
+- **New features:** No
+- **Enhancements:** Allowed
+- **Support length:**
+
+  - **Standard:** Approximately 6 months, until the next minor release is performed
+  - **Extended:** One year
+
+### Partial support
+
+At the end of the *full support* phase, a Silverstripe CMS version transitions to the *partial support* phase.
+
+- **Security fixes:** Partial – only high and critical impact security vulnerabilities (those with a CVSS score of 7.0 or above) are addressed
+- **Bug fixes:** No
+- **New features:** No
+- **Enhancements:** No
+- **Support length:**
+
+  - **Standard:** 6 months
+  - **Extended:** Approximately one year, extending until the following subsequent major release (e.g. CMS 5.4 will remain in partial support until CMS 7.0 is released)
```

Comment:
```suggestion
  - **Extended:** Approximately one year, extending until two subsequent major versions have been released (e.g. CMS 5.4 will remain in partial support until CMS 7.0 is released)
```
Just makes this a little clearer

Reply:
Could also be "until two subsequent major releases have been performed" which has the same effect.

Reply:
Updated